README.md

```diff
@@ -424,6 +425,21 @@ list StringList {
   member: String
 }
 ```
+## Protocol Compliance Module
+ - Alloy contains a suite of protocol tests utilizing the [AWS HTTP Protocol Compliance Test Module]("https://smithy.io/2.0/additional-specs/http-protocol-compliance-tests.html)
+ - These can be used to test an implementation of the simpleRestJson protocol to confirm compliance with the protocol .
+   For sbt:
+
+### Using the Protocol Compliance Tests
+```scala
+"com.disneystreaming.alloy" %% "alloy-protocol-tests" % "x.x.x"
```

Comment:
```suggestion
"com.disneystreaming.alloy" % "alloy-protocol-tests" % "x.x.x"
```

Reply:
lol , yes , I changed it because it was going to become a Scala module , forgot to switch it back, thanks for catching